net-misc/youtube-viewer/youtube-viewer-3.7.4-r1.ebuild

```diff
@@ -80,6 +80,10 @@ pkg_postinst() {
 	elog
 	elog "Check the configuration file in ~/.config/youtube-viewer/"
 	elog "and configure your video player backend."
+	elog
+	ewarn "Starting with version 3.7.4, youtube-viewer requires the user"
+	ewarn "to get their own API key to function. Please refer to README.md"
+	ewarn "for further details!"
```

Comment:
You could also add a link to upstream doc about it, since it's super easy to just click a link on your terminal for instructions.

Also please reduce the character amount in your git commit summary, it should be 70 chars long at maximum. Its currently 74. You can utilize to git message box to put more information there. So like, 
```
net-misc/youtube-viewer: Add ewarn message about API requirement 

- Starting with version 3.7.4, youtube-viewer requires the user
  to get their own API key to function.

Package-Manager: Portage-2.3.93, Repoman-2.3.20
Signed-off-by: Zoltan Puskas <zoltan@sinustrom.info>
```

Reply:
README.md is installed with the package into "/usr/share/doc/youtube-viewer-3.7.4/README.md.bz2" but sure I can also add a link...